18F/identity-idp
# Clear failure cookie on component load

## 🎫 Ticket

https://cm-jira.usa.gov/browse/LG-12155

## 🛠 Summary of changes

This is a small change for an edge case that I ran into while working in this area of the code base. Basically, we now proactively wipe a cookie when we load the page.

Worth doing this because if this is happening for real users, then it would prevent them from accessing the acuant SDK unless they cleared their cookies.

## 📜 Testing Plan

I wasn't able to come up with a way to reliably test. But to confirm you would
- Make sure the acuant fail cookie was present (not cleared).
- Load the upload (front/back/selfie) page.
- Note that before this PR the Acuant SDK won't work, you'll get the other options.
- After this PR you'll get the Acuant SDK because the cookie gets cleared on page load.
